pkgs/sdk_triage_bot/lib/triage.dart

```diff
@@ -46,6 +46,22 @@ Future<void> triage(
   }
   logger.log('');
 
+  // If the issue has any comments, retrieve and include the last comment in the
+  // prompt.
+  String? lastComment;
+  if (issue.hasComments) {
+    final comments = await githubService.fetchIssueComments(sdkSlug, issue);
+    final comment = comments.last;
+
+    lastComment = '''
+---
+
+Last comment by @${comment.user?.login}:
```

Comment:
I don't know the first thing about prompting, but maybe add some more instructions, similar to what you wrote in this PR description? Something like `This comment might contain helpful information for triaging, but if not, ignore it...` or similar?

Reply:
sgtm; I'll word-smith this a bit.